src/hooks/listHooks.js

```diff
@@ -102,7 +102,18 @@ export const useRenderItem = (query, navigation, options = {}) => {
           return <EventSectionHeader {...{ item, navigation, options, query }} />;
         }
 
-        if (index === 0) {
+        if (index === 0 || (query === QUERY_TYPES.EVENT_RECORDS && index === 1)) {
```

Comment:
i don't understand the case for the second entry in a event list having this special part? what about index 0 or 2 or 3?

Reply:
There is a special case in `EventList`. In EventList, there is a `SectionHeader` where we show the dates. `SectionHeader` is the 0th index of the list. If we want the first item to have a big picture, the first item shown for `EventList` is the item at index 1. That's why I added a logic in this way. I can explain this with the comment line.

Reply:
Thanks, now i got it! You explained it well, please add it as a comment in the code, yes.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/981/commits/3965aa1441ec9e3a14e581d5a80b6694310bc142